modules/sdk-lib-mpc/test/unit/schnorrProof.ts

```diff
@@ -0,0 +1,34 @@
+import 'should';
```

Comment:
We need to deprecate `should`. It cannot handle `BigInt`: `({a: 1n}).should.eql({a: 2n})` will pass.